.github/workflows/ci.yml

```diff
@@ -812,6 +812,15 @@ jobs:
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-a2-highgpu-1g.json" \
               --compile_stats_files="${COMPILE_STATS_RESULTS}"
 
+  benchmark_db_import:
+    needs: [setup, compilation_benchmarks, execution_benchmarks]
+    if: fromJson(needs.setup.outputs.should-run) && needs.setup.outputs.benchmark-presets != ''
+    uses: ./.github/workflows/benchmark_db_import.yml
```

Comment:
Since this workflow is relatively simple. I'm wondering if we should write it as a script for reusability and run it in the existing jobs (e.g. `process_benchmark_results`) instead of spawning a new one (which consumes more resources)

Reply:
SGTM. Where do you think should I place this script?

Reply:
I just realized I could also make it an action instead of a callable workflow. Then it would just be a step in an existing job.

Reply:
SGTM

Reply:
I now moved the logic into a bash script. Let me know what you think.